test/test_upgrade_database.cpp

```diff
@@ -1211,4 +1212,62 @@ TEST_IF(Upgrade_Database_6_7, REALM_MAX_BPNODE_SIZE == 4 || REALM_MAX_BPNODE_SIZ
 #endif // TEST_READ_UPGRADE_MODE
 }
 
+TEST_IF(Upgrade_Database_7_8, REALM_MAX_BPNODE_SIZE == 4 || REALM_MAX_BPNODE_SIZE == 1000)
```

Comment:
It would be awesome if you could add a test that a the version "5_to_6.realm" file can be opened in read-only mode without requiring any upgrade (checking that the format it still version 5 and not version 8). I'm not sure how to write a test like this that will be future proof when we increase the version again but it would be nice to have even temporarily.